lib/bolt/puppetdb/config.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'json'
 require 'bolt/util'
+require 'win32/dir'
```

Comment:
I think this require (and the `Dir::COMMON_APPDATA` reference) will only work when running on Windows. So we probably need to guard that, which might mean reorganizing the data structures below a bit.